Directory.Build.targets

```diff
@@ -20,11 +20,11 @@
     <ItemGroup>
       <InputAssemblies
         Include="$(TargetDir)\*.dll"
-        Exclude="$(TargetPath)" />
+        Exclude="$(TargetPath);$(TargetDir)\Microsoft.*.dll;$(TargetDir)\System.*.dll;" />
```

Comment:
ILRepack was adding other assemblies to SkyLib which caused a ReflectionLoadTypeException crash.